### ISSUE TITLE
New database not showing up in Web UI

### ISSUE THREAD

**Issue by aznwarmonkey** [arangodb/arangodb-java-driver#93]
I currently am running arango v 3.1.8 in cluster mode.  From the java driver i created the database which and returns true after call but the web UI is not registering the changes.  If I run the code again, it states the database already exists.   I believe I see the same issue  with collections as well.

Is this a bug or user error?  If user error what am I doing wrong?

**Comment by mpv1989** [comment]
Thanks for reporting.
This is a bug in the webUI and not in the java driver. I contacted a colleague who will fix it.

**Comment by hkernbach** [comment]
Fixed. Will be included in next bugfix release.

**Comment by SimonCanJer** [comment]
I have installed 3.6.5 for Windows as a trial to select db for the project. I use Java API together with spring. Users are managed OK.
But then goes something out of my comprehension, I create and persist databases, collections, documents in Java API. It works. But I see no one of the created objects in Web UI (http://127.0.0.1:8529/). Furthermore, when I delete a database over the UI, Java API continues returning the database with the same name. Seems API and WEB UI create objects in parallel.   I was intended to demonstrate a POC  but really frustrated now and will replace it with another engine. What happens, does somebody mind?

**Comment by rashtao** [comment]
@SimonCanJer The Java driver and the web UI perform requests to the same db API. You can further investigate checking the generated requests of the Web UI in the network monitor of your browser and the ones from the Java driver, enabling `debug` log levels globally.

**Comment by SimonCanJer** [comment]
Thanks, I got that I did wrong with UI. However, I must every time update
UI page of the collection in order to see changes, which are done way of
API. At the same time, maybe I can advice me with one really
impeding issue, that I see in Java API. I have db. query("FOR t in "+
collection+" return t",BaseDocument.class);
It is ok, when storage is an alphabetic string. But if it is something like
"of-com-dto-MyClass" it does a split by "\\-" and takes the part after last
"-". "-" described as a legal character... I can avoid the problem simply
by another naming convention, but why it is?
Thanks, Simon
